include/provider/data/data_transaction.h

```diff
@@ -0,0 +1,33 @@
+#ifndef ERD_PROXY_DATA_TRANSACTION_H
+#define ERD_PROXY_DATA_TRANSACTION_H
+
+#include <string>
+
+struct TransactionHash
+{
+    std::string hash;
+};
+
+class TransactionStatus
```

Comment:
Perhaps also add a comment which references the `erdjs` implementation (e.g. "Also see ...")?

Reply:
Added comment